### PR TITLE
fix: updating NOTES.txt in helm charts

### PR DESCRIPTION
**Pull request by hferentschik**
- aligning some wording across docs
- deleting obsolete README
- updating .gitignore

fixes issue #75

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,14 +15,19 @@ $ jx add app jx-app-jacoco --repository "http://chartmuseum.jenkins-x.io"
 
 NOTE: The syntax of this command is evolving and will change.
 
-Upon successful installation, you should see jx-app-jacoco in the list of pods (`kubectl get pods`) running in your cluster - it will be called `jx-app-jacoco-jx-app-jacoco`.
+After the installation, you can view the status of jx-app-jacoco via:
+
+```bash
+$ helm status jx-app-jacoco
```
Comment (pmuir) [inline README.md:21]: Add a note here that this is a workaround until we implement `jx get apps`

Comment (hferentschik) [reply README.md:21]: Done

**Pull request merged**